docs/cli.rst

```diff
@@ -211,6 +211,17 @@ The VHDL revision can be specified through the :ref:`python_interface`
 Alternatively, environment variable ``VUNIT_VHDL_STANDARD`` can be set to
 ``1993``, ``2002``, ``2008`` or ``2019``.
 
+.. note:: ``93``, ``02``, ``08`` or ``19`` can be also used.
+
+.. note:: ``2008`` is the default.
+
+.. note:: you will receive an ERROR if you specify a not supported revision
```

Comment:
I'm not sure this is required. In general, VUnit will report errors adequately when found. I mean, this is not specific to this feature, but a characteristic of the project.

Reply:
Ok... Was annoying to receive an error when I tried a valid standard... Maybe clarify that the supported standard depends on the EDA tools used, and an ERROR is issued when not supported... I didn't found a section indicating which standards are supported by which tools. A link to this info would be enough...